### PR TITLE
Add bernoulli primitive op and support dropout op in new AD.

### PR DESCRIPTION
**Pull request by levi131**
### PR types
New features

### PR changes
OPs

### Describe
Add `bernoulli_p` autograd primitive op and support orig2prim for paddle original `dropout` op.

### CODE REVIEW

File: python/paddle/incubate/autograd/primrules.py [hunk 1]
```diff
@@ -420,6 +421,31 @@ def gelu_orig2prim(op, x):
                 erf(mul(x, fill_const(1 / math.sqrt(2.), x.shape, x.dtype)))))
 
 
+@REGISTER_ORIG2PRIM('dropout')
+def dropout_orig2prim(op, x, seed_t=None):
+    assert seed_t is None, 'Can not lower concat into prim ops with seedtensor.'
+    mask = bernoulli(shape=x.shape, dtype=x.dtype, p=op.attr('dropout_prob'))
```
Comment (cxxly) [inline python/paddle/incubate/autograd/primrules.py:446]: 'Can not lower concat into prim ops with seedtensor.' 中concat是否为拼写错误？

**Comment by cxxly** [comment]
请增加dropout至少一阶反向测试，可以通过固定seed进行数值对齐

File: python/paddle/incubate/autograd/primrules.py (hunk 1 above)
Comment (levi131) [reply python/paddle/incubate/autograd/primrules.py:446]: 已修复

**Review by cxxly — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**